cypress/e2e/header.spec.cy.ts

```diff
@@ -0,0 +1,128 @@
+import {YasguiSteps} from '../steps/yasgui-steps';
+import {YasqeSteps} from '../steps/yasqe-steps';
+import {YasrSteps} from '../steps/yasr-steps';
+import {QueryStubs} from "../stubs/query-stubs";
+import {HeaderPageSteps} from '../steps/header-page-steps';
+
+describe('View modes', () => {
+    beforeEach(() => {
+        QueryStubs.stubDefaultQueryResponse();
+    });
+
+    it('Should headers with render mode button be visible', () => {
+        // When I visit page with ontotext-yasgui
+        HeaderPageSteps.visit();
+
+        // I expect header with buttons to be visible If configuration "showHeader" is undefined
+        cy.get('.ontotext-yasgui-header').should('be.visible');
```

Comment:
Move any access and interraction with the DOM inside the steps. This includes selectors like this, click actions and other mouse activities, etc. I don't want the tests to be tied with the implementation

Reply:
done